jobs-page-styling.css

```diff
@@ -0,0 +1,184 @@
+
+* {
+    margin: 0;
+    padding: 0;
+    font-family: 'poppins';
+    box-sizing: border-box;
+}
+
+h1 {
+    font-family: "Sansita Swashed",system-ui ;
+    margin-top: 2rem;
+    font-size: 30px;
+}
+@keyframes appear {
+    from {
+        opacity: 0;
+        clip-path: inset(100% 100% 0 0);
+        to {
+            opacity: 1;
+            clip-path: inset(0 0 0 0);
+            
+        }
+    }
+    
+}
+
+.container { 
+    background-color: #ECFCFE;
+}
+
+.jobs-section {
+
+    display: flex;
+    flex-direction: column;
+    justify-content: center;
+    align-items: center;
+    background-color: #E4F2FF;
+
+}
+
+.offers {
+    margin-top: 2.775rem;
+    width: 90%;
+    margin-left: 5%;
+    display: flex;
+    justify-content: space-between;
+    gap: 10px 10px;
+}
+
+.offerCarte  {
+
+    animation: appear linear;
+    animation-timeline: view();
+    animation-range:entry 0% cover 40%;
+
+    background-color: #ffffff;
+    width: 50%;
+    height: auto;
+    box-shadow: 2px 2px #626970;
+}
+ button {
+    width: 70%;
+    height: 22px;
+    transition: 0.6s;
+    font-size: 15px;
+    background-color:#365ED0;
+    margin-left: 15% ;
+    margin-bottom:3% ;
+    border:none;
+}
+button:hover {
+    cursor:pointer;
+    box-shadow: 3px 3px #626970;
+    transform:scale(1.01);
+    transition:0.6s;
+
+}
+a {
+    color:white;
+}
+.location {
+    padding-left: 5px;
+    display: flex;
+    width: 70%;;
+    justify-content: space-evenly;
+}
+.location i {
+    margin-top: 4px;
+    padding-right: 5px;
+}
+.offerCarte img {
+    width:30px;
+}
+.company {
+    display: flex;
+    margin:5px;
+}
+.company > div p:first-child{
+    margin-left: .5rem;
+    font-size: 1rem;
+    color: initial;
+    font-weight: bold; 
+}
+.company > div p:nth-child(2)
+{
+    margin-left: 5px;
+    font-size: 13px;
+}
+@media screen and (max-width:700px) {
+    .offers {
+        display: grid;
+        grid-template-areas: " c1r1 c2r1 " " c1r2 c2r2 ";
+    }
+    .offerCarte {
+        width: 100%;
+    }
+}
+@media screen and (max-width:500px) {
```

Comment:
remove this media query because we need just one media query for version desktop